docs/en/setup/backend/backend-receivers.md

```diff
@@ -128,3 +129,20 @@ receiver-oc:
     gRPCHost: ${SW_OC_RECEIVER_GRPC_HOST:0.0.0.0}
     gRPCPort: ${SW_OC_RECEIVER_GRPC_PORT:55678}
 ```
+
+## Kafka receiver
+
+Kafka Receiver supports to receive `TraceSegment`, `JVMMetrics` and others of Agent reported as gRPC Receiver.
```

Comment:
Trace and JVM Metrics, please use common nouns. Don't use the class name in the codes.